packages/@react-spectrum/menu/chromatic/MenuTrigger.chromatic.tsx

```diff
@@ -91,6 +91,12 @@ let withSection = [
   ]}
 ];
 
+let withArabic = [
+  {name: 'ينسخ', icon: 'Copy', shortcut: '⌘ن'},
+  {name: 'يقطع', icon: 'Cut', shortcut: '⌘C'},
+  {name: 'معجون', icon: 'Paste', shortcut: '⌘V'}
```

Comment:
```suggestion
  {name: 'نسخ', icon: 'Copy', shortcut: '⌘C'},
  {name: 'قص', icon: 'Cut', shortcut: '⌘X'},
  {name: 'لصق', icon: 'Paste', shortcut: '⌘V'}
```
Reference: 
![image](https://user-images.githubusercontent.com/11140840/128249635-02957b8c-3043-4969-97f8-91c80b575ee8.png)